values.yaml

```diff
@@ -544,6 +544,11 @@ server:
     successThreshold: 1
     # Number of seconds after which the probe times out.
     timeoutSeconds: 3
+    # Used to define a liveness command
+    execCommand: []
+    # - /bin/sh
+    # - -c
+    # - /vault/userconfig/mylivenessscript/run.sh
```

Comment:
I'd suggest moving this up by `path`, and mention the preference order in the comment.

Reply:
Thank you. Will do!